tests/test_util.py

```diff
@@ -66,27 +67,109 @@ def test_clone_git_repo_raises_explicit_exception_on_clone_fail(
 
 
 class OutputTests(unittest.TestCase):
+    @mock.patch("tartufo.scanner.ScannerBase")
+    @mock.patch("tartufo.types.GlobalOptions")
```

Comment:
Not a blocker, but I actually built out a helper so that you don't have to mock options. In this case you would be able to call:

```python
from tests.helpers import generate_options

# ...
options = generate_options(GlobalOptions, json=True, verbose=0)
```

I tend to lean toward as little mocking as possible. But again, not a blocker. 😄

Reply:
Updated to use the testing helper for options. Nice.